### PR TITLE
CLN: remove python2 file encodings

### PR DESCRIPTION
**Pull request by topper-123**
- [x] xref #25725

Remove usage of ``# coding=utf-8`` pattern from code base.

### CODE REVIEW

File: pandas/tests/util/test_assert_almost_equal.py [hunk 1]
```diff
@@ -1,4 +1,3 @@
-# -*- coding: utf-8 -*-
```
Comment (WillAyd) [inline pandas/tests/util/test_assert_almost_equal.py]: Probably worth deleting leading whitespace (applicable in a few modules)

**Review by WillAyd — COMMENTED** [review]
OK by me. For Cython it looks like default is UTF8 as well:

http://docs.cython.org/en/latest/src/tutorial/strings.html#source-code-encoding

Should be OK to remove from any file ending in .pyx or .pxi.in

File: pandas/tests/util/test_assert_almost_equal.py (hunk 1 above)
Comment (topper-123) [reply pandas/tests/util/test_assert_almost_equal.py]: Ok, slightly confusing comment coming:

I saw that just after pushing, and was working with ``-# coding=utf-8`` at the time, so I changed this PR to deal with that (including dealing with whitespace).

**Comment by jreback** [comment]
thanks @topper-123

**Pull request merged**